labelbox/schema/export_task.py

```diff
@@ -554,15 +554,19 @@ def _get_metadata_header(
 
     def get_total_file_size(self, stream_type: StreamType) -> Union[int, None]:
         """Returns the total file size for a specific task."""
-        if not self._task.status in ["COMPLETE", "FAILED"]:
+        if self._task.status == "FAILED":
+            raise ExportTask.TaskNotReadyException("Task failed")
```

Comment:
yeah, while I was adding that, I also thought that the name was inappropriate for this specific situation. I was just reusing the same exception I had at hand. I will rename it to `ExportTaskException`, this way it will be more generic. Thanks!